keyboards/bm68rgb/.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This should be removed.

Reply:
`git rm --cached -- keyboards/bm68rgb/.vscode/c_cpp_properties.json` followed by `git commit` will remove it from the branch, but not your PC. Omit the `--cached` argument if you want to remove the file from your system as well.

Reply:
I did a git add instead of a git add -u, fun!
I now know about --cached. Time to read the man page